esmvalcore/cmor/_fixes/cmip6/ec_earth3.py

```diff
@@ -75,3 +75,35 @@ def fix_metadata(self, cubes):
             latitude.bounds = np.round(latitude.core_bounds(), 8)
 
         return cubes
+
+
+class Oh(Fix):
+    """Fixes for oh."""
+
+    def fix_metadata(self, cubes):
+        """Fix standard name for ps.
+
+        Fix standard_name for Surface Air Pressure (ps).
+        See discussion in
+        https://github.com/ESMValGroup/ESMValCore/issues/2613
+        Cube has two coordinates called air_pressure: an AuxCoord ps
+        and a DerivedCoord that is 4D and derived using formula terms,
+        we are setting the former's standard_name to None.
+
+        Parameters
+        ----------
+        cubes : iris.cube.CubeList
+            Input cubes.
+
+        Returns
+        -------
+        iris.cube.CubeList
+        """
+        cube = self.get_cube_from_list(cubes)
+
+        for cube in cubes:
+            for coord in cube.coords():
+                if coord.var_name == "ps":
+                    coord.standard_name = None
```

Comment:
```suggestion
                    coord.standard_name = "surface_air_pressure|
```
according to the CF Conventions: https://cfconventions.org/cf-conventions/cf-conventions.html#_atmosphere_hybrid_sigma_pressure_coordinate

Reply:
@schlunma argues `None` and I haven't the foggiest - settle it, gents :grin:

Reply:
Bouwe's right 🍻